Update embed.md (#3603)
nasa-gibs/worldview: doc/embed.md

# Embedding Worldview

> 

Reviewer: ```suggestion

## Create an Embedded Worldview
```

3. Click on the Share icon (second from the left) in the upper right corner.
4. Click on the "Embed" tab.

> 5. Click "Copy" to copy the HTML code to wrap the current Worldview instance in an `<iframe>` inline element.

Reviewer: ```suggestion
5. Click "Copy" to copy the HTML code with the current Worldview instance in an embeddable `<iframe>` inline element.
```

6. Add the HTML code to your webpage in the place where you want the Worldview map to appear.

> Note: You will need to wrap the `<iframe>` code in a wrapper element (e.g., `<div>`) with your desired width, height, and any additional style changes. You can change the width and height from percent to px to make it fit your page.

Reviewer: ```suggestion
Note: You will need to wrap the `<iframe>` code in a wrapper element (e.g., `<div>`) with your desired width, height, and any additional style changes. You can change the width and height from percent to px to make it fit your page. See [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) for more information on the `<iframe>` inline element including an explanation of its attributes.
```

> See [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) for more information on the `<iframe>` inline element including an explanation of its attributes.

Reviewer: ```suggestion
```

**Example `<iframe>` code:**

> 

Reviewer: ```suggestion
```

```<iframe src="https://worldview.earthdata.nasa.gov/?em=true" role="application" sandbox="allow-modals allow-scripts allow-same-origin allow-forms allow-popups" width="100%" height="100%" allow="fullscreen; autoplay;" loading="lazy"></iframe>```

> **Notes:** Compared to the full application, the embedded version of Worldview has intentionally limited functionality to give users a more streamlined experience. The following is a list of the features and button/tab interactions that are not available or altered in embed mode.

Reviewer: ```suggestion
## Limited Features in an Embedded Worldview

Compared to the full application, the embedded version of Worldview has intentionally limited functionality to give users a more streamlined experience. The following is a list of the features and button/tab interactions that are not available or altered in embed mode.
```

Reviewer 2: `The following is a list of the features and button/tab interactions that are not available or altered in embed mode.` -- I'd suggest replacing the `.` with a `:`.

> - Search places by location, Share this map, Switch projections, Take a snapshot, and the Information button are disabled. These features have been replaced with a button that brings user back to the full version of Worldview showing the embedded map contents.

Reviewer: ```suggestion
- Search places by location, Share this map, Switch projections, Take a snapshot, and Information buttons are disabled. These features have been replaced with a button that opens up a new tab with your current map view in the full featured Worldview.
```

- The welcome page and its tour stories are disabled.
- In the Layer List, layer grouping is disabled, the hidden layers are not displayed, and the options to change the opacity, thresholds, color palettes, disable/enable classifications, or other related settings are not available.

> - The Events tab will only appear in the embedded version if it is selected in Worldview before creating the iframe html code.

Reviewer: ```suggestion
- The Events tab will only appear in the embedded version if both 1) the tab is selected and 2) an event is selected in Worldview before creating the `iframe` html code.
```

> - The Events tab will only show the event that was initially selected in Worldview.

Reviewer: ```suggestion
```

Author: ✔️

> - Event tab options are disabled.

Reviewer: ```suggestion
```

- Data Download is disabled.
- The Extended timeline and layer coverage panel are unavailable. However, it is still possible for users to change the date.
- The Export to Animation tool is disabled.
- The Measure Tool is disabled.

> Helpful Tips when embedding:

Reviewer: ```suggestion
## Helpful Tips

```
